app/tests/integration/nimbus/test_e2e_create_experiment.py

```diff
@@ -228,3 +228,62 @@ def test_create_new_experiment_remote_settings_reject(selenium, base_url):
             selenium.refresh()
     else:
         raise AssertionError("Experiment page didn't load")
+
+
+def test_create_new_experiment_remote_settings_timeout(selenium, base_url):
+    experiment_name = f"name here remote {random.randint(0, 1000)}"
+
+    selenium.get(base_url)
+    home = HomePage(selenium, base_url).wait_for_page_to_load()
+    # home.tabs[-1].click()  # Click drafts
+    # current_experiments = None
+    # try:
+    #     current_experiments = len(home.tables[0].experiments)
+    # except TimeoutException:
+    #     current_experiments = 0
+    experiment = home.create_new_button()
+    experiment.public_name = experiment_name
+    experiment.hypothesis = "smart stuff here"
+    experiment.application = "DESKTOP"
+
+    # Fill Overview Page
+    overview = experiment.save_and_continue()
+    overview.public_description = "description stuff"
+    overview.select_risk_brand_false()
+    overview.select_risk_revenue_false()
+    overview.select_risk_partner_false()
+
+    # Fill Branches page
+    branches = overview.save_and_continue()
+    branches.remove_branch()
+    branches.reference_branch_name = "name 1"
+    branches.reference_branch_description = "a nice experiment"
+    branches.feature_config = "No Feature Firefox Desktop"
+
+    # Fill Metrics page
+    metrics = branches.save_and_continue()
+
+    # Fill Audience page
+    audience = metrics.save_and_continue()
+    audience.channel = "Nightly"
+    audience.min_version = 80
+    audience.targeting = "US_ONLY"
+    audience.percentage = 50.0
+    audience.expected_clients = 50
+    audience.save_btn()
+    review = audience.save_and_continue()
```

Comment:
Unrelated to this PR, so feel free to swiftly ignore this... But wondering if this is very boilerplatey and we want to eventually just have some ridiculous "make_fake_experiment()" function which does all this repetitive heavy lifting. Although I guess that means all the experiments would be boilerplate and hard to change specific things like "channel" for a specific test (unless we can easily override certain fields). I'd have to revisit the previous PR and see how much of this is similar.

Or, possibly that's a terrible idea, and I should be shunned by the community.

Reply:
> 
> 
> Unrelated to this PR, so feel free to swiftly ignore this... But wondering if this is very boilerplatey and we want to eventually just have some ridiculous "make_fake_experiment()" function which does all this repetitive heavy lifting. Although I guess that means all the experiments would be boilerplate and hard to change specific things like "channel" for a specific test (unless we can easily override certain fields). I'd have to revisit the previous PR and see how much of this is similar.
> 
> Or, possibly that's a terrible idea, and I should be shunned by the community.

No this is great. It is the first thing I am doing next quarter before we add more tests.